README.md

```diff
@@ -2,7 +2,8 @@
 
 [![Build Status](https://travis-ci.org/ioos/compliance-checker.svg)](https://travis-ci.org/ioos/compliance-checker)
 
-The IOOS Compliance Checker is a Python tool to check local/remote datasets against a variety of compliance standards. It is primarily a command-line tool (tested on OSX/Linux) and can also be used as a library import.
+The IOOS Compliance Checker is a Python tool to check local/remote datasets against a variety of compliance standards.
+It is primarily a command-line tool (tested on OS X/Linux) and can also be used as a library import.
```

Comment:
@lukecampbell I added a few line breaks for readability. If that scrambles the text too much let me know and I will remove this change.